docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   organizationName: "jitsi",
   projectName: "handbook",
   favicon: "img/favicon.png",
-  onBrokenLinks: "throw",
```

Comment:
please don't change this

Reply:
My purpose in changing this keyword was, to not get errors blocking the code when running it.  So I changed "throw" to "log" so that all the errors are logged.

Reply:
@saghul 
I have fixed this code and I am making going to make commits now